isis/appdata/translations/pds4ExportPolarStereographic.trn

```diff
@@ -248,13 +248,12 @@ End_Group
 
 Group = StraightVerticalLongitudeFromPole
   Auto
-  InputKey       = CenterLatitude
+  InputKey       = CenterLongitude
   InputGroup     = "IsisCube,Mapping"
   InputPosition  = (IsisCube, Mapping)
   OutputName     = cart:straight_vertical_longitude_from_pole
   OutputPosition = (Product_Observational, Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Map_Projection, cart:Polar_Stereographic)
-  Translation    = (180, 90.0)
-  Translation    = (0, -90.0)
+  Translation    = (*, *)
```

Comment:
Sorry, can you explain this change?

Reply:
I am believe it just takes whatever was passed in and passes that same value out. 

I am a little curious why there is no longer a need for a longitude domain reconciliation, but I am guessing that since a Polar Stereographic projection contains all longitudes, as long as the user is consistent, it will resolve. I assumed this and (possibly prematurely) approved the changes. If you could verify or correct me, that would be great!

Reply:
PDS4 Cartography dictionary (thus PDS4 labels) don't have the concept of Longitude Domain. For a map projection, it is "mathematically" unnecessary. Although if we feel it is necessary it can be added. https://github.com/pds-data-dictionaries/ldd-cart

Reply:
I am not sure I have enough knowledge to know if it is necessary in this context. However, If the PDS4 standard is set up without it and it is not mathematically necessary, I think it is fine. @Kelvinrr, do you disagree? If not, I am good to merge!

Reply:
Seems good to merge, thanks for the explanations!